helpers/test-data-generator.js

```diff
@@ -384,7 +384,7 @@ export async function setupRandomColony(colonyNetwork) {
 }
 
 export async function setupColony(colonyNetwork, tokenAddress) {
-  const { logs } = await colonyNetwork.createColony(tokenAddress, 0, "", "", false);
+  const { logs } = await colonyNetwork.createColony(tokenAddress, 0, "", "", true);
```

Comment:
Not sure what the point of this particular change is?

Reply:
Gosh me neither.